packages/snjs/lib/application.ts

```diff
@@ -888,6 +888,22 @@ export class SNApplication {
     return this.protocolService.hasAccount();
   }
 
+  /**
+   * Use this method if you do not want to display protections-related
+   * information to a user who cannot currently exploit the feature.
+   */
+  public hasProtectionSources(): boolean {
```

Comment:
What do you think of a comment like this:

```
Returns true if the user has a source of protection available, such as a passcode, password, or biometrics.
```

I find the current comment somewhat awkwardly phrased. "Use..if you do not..who cannot..exploit"..

Reply:
Sounds good! I was struggling with this one